components/dropdown/dropdown-menu.js

```diff
@@ -80,6 +81,10 @@ class DropdownMenu extends DropdownContentMixin(LitElement) {
 		this.close();
 	}
 
+	_onFocus() {
+		this.__getMenuElement().focus();
```

Comment:
Won't this move focus to the menu when the time selector receives focus? I would have thought we'd want to focus on the text input... but maybe I'm misunderstanding something.

Reply:
This applies to the `d2l-dropdown-menu` element (here: https://github.com/BrightspaceUI/core/blob/master/components/inputs/input-time.js#L162) which is more like the content wrapper/container for the dropdown box and doesn't include the text input. When you tab, it moves the focus from the dropdown opener (the text input) to that wrapper, but without this code it wasn't shifting from the wrapper to the actual menu (thus the 'blank' tab stop). Hopefully that makes sense.

Reply:
Ah gotcha, makes sense -- thanks!

Reply:
I think the original intent of the code that creates the tab-stop on the container was - for generic (non-menu) content, we need to move the focus into the dropdown _somewhere_, so if there are no focusable elements in the content, we create that tab stop on the fly.  Not ideal... a corner case though. In the `d2l-dropdown-menu` case, this was thought to never be hit because we can theoretically expect there to be a focusable element, the menu/items.  So... I'm wondering why this usage is different?

Reply:
I was able to reproduce the bug using a generic dropdown menu with enough items to require scrolling, so I think it just went under the radar because most menus aren't this long. I don't know why exactly it only kicked in when it reached scrolling length, though.